target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mf286d.dts

```diff
@@ -0,0 +1,462 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2022, Pawel Dembicki <paweldembicki@gmail.com>.
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "ZTE MF286D";
+	compatible = "zte,mf286d";
+
+	reserved-memory {
```

Comment:
The "reserved-memory" node that is in upstream [5.10](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/arch/arm/boot/dts/qcom-ipq4019.dtsi?h=v5.10.95#n20) should be good enough. (And gives you more RAM.)
you should be able to just delete it.

Reply:
I droped it.